### PR TITLE
Fix compatibility with fastify-jwt v4

### LINKED ISSUE #171: Fix compatibility issues w fastify-jwt v4
The `dependabot` tried to update the library to use the latest version of `fastify-jwt` v4 but was unsuccessful. The main issue is that the token verification process is not compatible with the `fast-jwt` `verify` method. 

This issue needs to be investigated if it is possible to solve within this library or the fix needs to be done on `fast-jwt`.

If the issue is fixed in `fast-jwt` library that will require release of `fastify-jwt` as well.

### PR DESCRIPTION
**Pull request by radomird**
Closes #171 
- update tests
- fix verify options that are updated with v4

### CODE REVIEW

**Comment by radomird** [comment]
Once `fast-jwt` and then `fastify-jwt` are released, then we need to update the package.json and then this PR will be ready.

**Comment by simoneb** [comment]
let's use release automation for fast-jwt to release a new version and set it up in this repo too

**Comment by radomird** [comment]
Ready for review.

File: package.json [hunk 1]
```diff
@@ -61,9 +61,11 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-promise": "^5.1.0",
     "eslint-plugin-standard": "^5.0.0",
+    "fast-jwt": "^1.4.0",
     "fastify": "^3.1.1",
     "jest": "^27.0.3",
     "nock": "^13.0.2",
+    "path": "^0.12.7",
```
Comment (Eomm) [inline package.json:68]: Is it necessary?

Comment (radomird) [reply package.json:68]: I use it in the tests. But it's not a must more a convenience, if tests are moved at least we don't need to update the path to the keys.

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        required: false
+        default: "latest"
+  pull_request:
+    types: [closed]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: nearform/optic-release-automation-action@main
```
Comment (simoneb) [inline .github/workflows/release.yaml:21]: let's reference an actual version instead of main

File: package.json (hunk 1 above)
Comment (simoneb) [reply package.json:68]: path is a core module, you don't need to install it

**Review by mcollina — APPROVED** [review]
lgtm

File: package.json (hunk 1 above)
Comment (radomird) [reply package.json]: Removed it

File: test.js [hunk 3]
```diff
@@ -805,24 +901,24 @@ describe('General error handling', function () {
...
     })
   })
 
-  it('should complain if the JWT token has an unsupported algorithm', async function () {
-    const response = await server.inject({
-      method: 'GET',
-      url: '/verify',
-      headers: { Authorization: `Bearer ${tokens.unsupportedAlgorithm}` }
-    })
+  // it('should complain if the JWT token has an unsupported algorithm', async function () {
```
Comment (gdorsi) [inline test.js:908]: I don't know if it's ok, just reminding you that this test is commented

Comment (radomird) [reply test.js:908]: Good catch. Thanks.

**Pull request merged**